.github/workflows/turbo.yml

```diff
@@ -64,12 +61,16 @@ jobs:
                   aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
                   aws-region: us-east-2
 
+            - name: Install Doppler CLI
+              uses: dopplerhq/cli-action@v2
+
             - name: Build & test
-              run: yarn test:all
+              run: doppler run -- bash -c 'RENDER_PREVIEW=false yarn test:all'
```

Comment:
despite the render preview end, override `RENDER_PREVIEW=false` so that sourcemaps could be generated